<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="folder_path_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>300</y>
     <width>506</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Path To Slides</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="slide_selector_horizontalSlider">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>250</y>
     <width>196</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="slide_selector_comboBox">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>251</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select_slides_pushButton">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>297</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Folder...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoFile_save_pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>272</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_pushButton">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>297</y>
     <width>56</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="infoFile_select_pushButton">
   <property name="geometry">
    <rect>
     <x>515</x>
     <y>272</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Info File...</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoFile_Path_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>275</y>
     <width>506</width>
     <height>16</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Path To Info File</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>185</y>
     <width>306</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>Global Presentation Properties</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>306</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Page Properties</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>0</y>
     <width>321</width>
     <height>241</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>319</width>
      <height>239</height>
     </rect>
    </property>
    <widget class="QLabel" name="slide_viewer_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>320</width>
       <height>240</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
